wolfcrypt/src/ed448.c

```diff
@@ -368,13 +368,18 @@ int wc_ed448_sign_msg_ex(const byte* in, word32 inLen, byte* out,
 
         /* step 2: computing R = rB where rB is the scalar multiplication of
            r and B */
-        ge448_scalarmult_base(&R,nonce);
-        ge448_to_bytes(out,&R);
+        if (ret == 0) {
```

Comment:
No need to check this that I can see.

Reply:
Good point, it is redundant with check on line 364.

Reply:
Line 365 is returning.
Line 364 should check for '!= 0'.